core/proofs/impl/proof_engine_impl.cpp

```diff
@@ -8,12 +8,21 @@
 #include <filecoin-ffi/filcrypto.h>
 #include <thread>
 
+#include "codec/uvarint.hpp"
 #include "common/ffi.hpp"
-#include "primitives/address/address_codec.hpp"
 #include "primitives/cid/comm_cid.hpp"
 #include "proofs/proofs_error.hpp"
 #include "sector_storage/zerocomm/zerocomm.hpp"
 
+#define PROOFS_TRY(label)                                     \
```

Comment:
Maybe add variable as second parameter, otherwise MACRO requires 'res_ptr' variable

Reply:
Or rewrite it as a function.